### ISSUE TITLE
Localization cues

### ISSUE THREAD

**Issue by SII-Ferenas** [speedinghzl/DSRG#24]
HI, I cannot successfully install your module DRFI so I use another code by  (https://github.com/playerkk/drfi_matlab), and set the value < 0.06 as background seeds as what you write in the paper. However, I cannot reproduce the same results in your paper, the baseline (only with improved localization cues) in your paper is 52.5 mIou, but I only achieved 50.2% mIOU. btw, did the baseline you achieved not include the expand loss? what's the training hyperparamters in the Table 2? and Can you upload the localization cues with bg and fg please ? Thanks a lot

**Comment by speedinghzl** [comment]
The baseline does not include expand loss. The training hyperparameters are the same as DSRG (in this repo).  The [cues](https://drive.google.com/open?id=1cHyhjul9srPlwcl4xqrYR9MwzhFGwKXU) have been provided in this repo.

**Comment by SII-Ferenas** [comment]
Thanks a lot for your reply!!!

**Comment by TyroneLi** [comment]
> HI, I cannot successfully install your module DRFI so I use another code by (https://github.com/playerkk/drfi_matlab), and set the value < 0.06 as background seeds as what you write in the paper. However, I cannot reproduce the same results in your paper, the baseline (only with improved localization cues) in your paper is 52.5 mIou, but I only achieved 50.2% mIOU. btw, did the baseline you achieved not include the expand loss? what's the training hyperparamters in the Table 2? and Can you upload the localization cues with bg and fg please ? Thanks a lot

hI~could you share experiment details how to generate loc cue file as paper?I tried to reproduce, but my loc cue quality was so bad and cannot get a good segmentation performance. Hope your reply.
